leonardo-lfonzi62/week-6/monument-refactor-Homework/src/Blog/PostList /postList.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+
+class PostList extends Component {
+    render() {
+      return (
+        <section>
+
+		<h2>{this.params.title} </h2>
```

Comment:
I would work on cleaning up your code as you go. Spacing is a little off here and there.